qutip/tests/test_sesolve.py

```diff
@@ -352,3 +352,10 @@ def test_sesolve_bad_e_ops():
     tlist = np.linspace(0, 20, 200)
     with pytest.raises(TypeError) as exc:
         sesolve(H, psi0, tlist=tlist, e_ops=[qeye(3)])
+
+def test_sesolve_bad_H():
+    H = sigmaz(),
+    psi0 = basis(2, 0)
+    tlist = np.linspace(0, 20, 200)
+    with pytest.raises(TypeError) as exc:
+        sesolve(H, psi0, tlist=tlist, e_ops=[qeye(3)])
```

Comment:
```suggestion
        sesolve(H, psi0, tlist=tlist, e_ops=[qeye(3)])
    assert str(exc.value).startswith("Invalid H:")
```
I guess we should check that the right exception message is raised too.